src/dbnode/storage/bootstrap/bootstrapper/peers/source.go

```diff
@@ -185,6 +185,9 @@ func (s *peersSource) Read(
 			continue
 		}
 
+		// NB(bodu): We need to evict the info file cache before reading index data since we've
+		// maybe fetched blocks from peers so the cached info file state is now stale.
+		cache.Evict()
```

Comment:
Think we just want to evict before iterating all namespaces?

Reply:
Oh yeah, missed that this was in a loop over the namespaces. We definitely only want to to do this once.